package.json

```diff
@@ -13,8 +13,8 @@
     "dist"
   ],
   "scripts": {
-    "prebuild": "rimraf dist && rimraf ethers-v4 && rimraf ethers-v4-for-tests && patch-package",
-    "build": "npm run compile:contracts && npm run build:typechain && npm run compile:ts && copyfiles \"ethers-v4/**/*.d.ts\" dist",
+    "prebuild": "rimraf dist ethers && patch-package",
+    "build": "npm run compile:contracts && npm run build:typechain && npm run compile:ts && copyfiles build/contracts/**/*.json -u 1 dist",
```

Comment:
@JGiter I know this is merged already, but what was the rationale for the `-u 1`? Is seems this slices off a path off the bottom of the path, but it's to clear to me why we want to do that. https://github.com/calvinmetcalf/copyfiles#command-line

Reply:
@JGiter ah, I see, thanks. Hmm, maybe what we should do is add copying the types from the `ethers` dir like `iam-client-lib` is doing: https://github.com/energywebfoundation/iam-client-lib/blob/2586770e5a0f1c6986392b718f181f370bd5a8a4/package.json#L14